docs/extensions/conventions.md

```diff
@@ -35,7 +35,7 @@ Some extensions might not only create resources in the seed cluster itself but a
 However, there are no credentials for the shoot for every extension.
 
 Extensions are supposed to use [`ManagedResources`](../concepts/resource-manager.md#ManagedResource-controller) to manage resources in shoot clusters.
-gardenlet deploys gardener-resource-manager instances into all shoot control planes, that will reconcile `ManagedResources` without a specified class (`spec.class=null`) in shoot clusters.
+gardenlet deploys gardener-resource-manager instances into all shoot control planes, that will reconcile `ManagedResources` without a specified class (`spec.class=null`) in shoot clusters. Mind that Gardener acts on `ManagedResources` with the `origin=gardener` label. In order to prevent unwanted behavior extensions should omit the `origin` label or provide their own unique value for it when creating such resources.
```

Comment:
Nit: always using unique values is probably better than using an empty label value.

```suggestion
gardenlet deploys gardener-resource-manager instances into all shoot control planes, that will reconcile `ManagedResources` without a specified class (`spec.class=null`) in shoot clusters. Mind that Gardener acts on `ManagedResources` with the `origin=gardener` label. In order to prevent unwanted behavior extensions should provide their own unique value for the `origin` label when creating such resources.
```

Reply:
Thanks for the review! In the release note I have skipped the part that you can omit the `origin` label because I am referring to the concrete functions that accept it as a parameter. However when we talk about managed resources one can also use the `Create` function (or can even create them by other means) which does not require the `origin` label to be passed. This is why in the conventions file I have added the possibility to completely ignore it. By this I am trying to say the following:
- If you are using the `CreateForShoot` function then give it an unique value, but do not give it an empty value
- If you are using the `Create` function then you can decide if you want to add the label or now

Does this make sense? WDYT?

Reply:
That makes sense, good enough for me. Thanks for explaining your reasoning!
/approve